Emergency maintenance access now automatically activates during radiation storms.

PetMudstone opened the pull request:
## About The Pull Request

Makes emergency maintenance access automatically activate when the radiation storm event occurs. It will also automatically be disabled once the storm subsides after a minute so that people can have a good amount of time to leave maint. Thanks to Mr Melbert for helping me with the timers!

## Why It's Good For The Game

Only a few jobs have bridge access by default, and these jobs won't always be taken or be available to give emergency maint access. This change makes it so crews, particularly lower population ones, have a better chance of survival if no one with bridge access is available to help.

## Changelog
:cl:
add: Emergency maintenance access will now automatically be enabled in response to radiation storms.
/:cl:

PetMudstone committed:
Adding basic emergency maint access triggers to radiation storm.

PetMudstone committed:
Add timers for turning on and off emergency maint access

Thanks to mrmelbert for telling me how to use timers!

Zonespace27 commented:
only potential nitpick i have is that it'll revoke maint access even if it was already enabled before the radstorm, wouldn't be hard to do a check for `GLOB.emergency_access` on both sets of code, though

PetMudstone committed:
Checks for if emergency access was already enabled.

Thanks to Merlin1230 for telling me how to do the if statements and set variables.
